src/coreclr/vm/rejit.cpp

```diff
@@ -750,7 +750,7 @@ HRESULT ReJitManager::UpdateNativeInlinerActiveILVersions(
                     CodeVersionManager *pCodeVersionManager = pModule->GetCodeVersionManager();
                     CodeVersionManager::LockHolder codeVersioningLockHolder;
                     ILCodeVersion ilVersion = pCodeVersionManager->GetActiveILCodeVersion(inliner.m_module, inliner.m_methodDef);
-                    if (!ilVersion.HasDefaultIL())
+                    if (!ilVersion.HasDefaultIL() && !fIsRevert)
```

Comment:
Its a bit misleading that this function has an fIsRevert parameter because there is no path that reaches here where fIsRevert = TRUE. If you look at the callers to UpdateActiveILVersions, RequestRevert always passes flags = 0 which prevents reaching this function and RequestReJIT always passes fIsRevert=FALSE which is the only way execution could get here. It might be better to remove the fIsRevert parameter from this method and add asserts in the caller to clarify that fIsRevert is (currently) mutually exclusive with this inliner behavior.